breaking-front/src/pages/SignUp/SignUp.js

```diff
@@ -55,131 +63,166 @@ const SignUp = () => {
-            name="profileImg"
-            accept="image/*"
-            onChange={handleImageUploadPreview}
-          />
-          <Button type="button" onClick={() => imageInput.current.click()}>
-            이미지 추가
-          </Button>
-          <Button type="button" onClick={imageDeleteClick}>
-            기본 이미지로 변경
-          </Button>
-        </UserImageContainer>
-        <p>닉네임</p>
-        <Input
+      <Style.Form onSubmit={handleSubmit} onKeyDown={handleKeyDown}>
+        <Style.UserImageContainer>
+          <Style.UserImageLabel>
+            <Style.UserImageInput
+              type="file"
+              ref={imageInput}
+              name="profileImg"
+              accept="image/*"
+              onChange={handleImageUploadPreview}
+            />
+            <Style.UserImage src={imageSrc} alt="미리보기" />
+            프로필 추가
+          </Style.UserImageLabel>
+
+          <Style.XMarkIcon>
+            <XMark onClick={imageDeleteClick} />
+          </Style.XMarkIcon>
+        </Style.UserImageContainer>
+        <SignUpInput
+          type="text"
+          label="이름"
+          placeholder="이름"
+          name="realname"
+          value={realname}
+          onChange={handleChange}
+          autoFocus
+          required
+        />
+        <SignUpInput
           type="text"
+          label="닉네임"
+          placeholder="닉네임"
+          errorMessage={isValidMessage.nickname}
           name="nickname"
           value={nickname}
           onChange={handleChange}
           onBlur={() => {
-            IsValidNickname.mutate({ nickname });
+            REGEXP.NICKNAME.test(nickname)
+              ? IsValidNickname.mutate({ nickname })
+              : setIsValidMessage((message) => ({
+                  ...message,
+                  nickname: MESSAGE.SIGNUP.INVALID_NICKNAME,
+                }));
           }}
-          maxLength="8"
+          required
         />
-        <br />
-        <p>전화번호</p>
-        <Input
+        <SignUpInput
           type="text"
+          label="전화번호"
+          placeholder="전화번호"
+          errorMessage={isValidMessage.phoneNumber}
           name="phoneNumber"
           value={phoneNumber}
           onChange={handleChange}
           onBlur={() => {
-            IsValidPhoneNumber.mutate({ phoneNumber });
+            REGEXP.PHONENUMBER.test(phoneNumber)
+              ? IsValidPhoneNumber.mutate({ phoneNumber })
+              : setIsValidMessage((message) => ({
+                  ...message,
+                  phoneNumber: MESSAGE.SIGNUP.INVALID_PHONENUMBER,
+                }));
           }}
+          maxLength="11"
+          required
         />
-        <br />
-        <p>상태메시지</p>
-        <Input
+        <SignUpInput
+          type="email"
+          label="이메일"
+          placeholder="이메일"
+          errorMessage={isValidMessage.email}
+          name="email"
+          value={email}
+          onChange={handleChange}
+          onBlur={() => {
```

Comment:
```suggestion
const inputOnBlur = (REGEXP,api,type,errorMessage)=>{
  REGXP.test(type)?api.mutate({type}):setIsValidMessage( 이하생략 )

}
```
이런식으로 공통되는것 묶는건 어떤가요??